package.json

```diff
@@ -89,8 +89,6 @@
   },
   "devDependencies": {
     "@types/bech32": "^1.1.1",
-    "@types/bip32": "^2.0.0",
```

Comment:
Types are redundant as they are already bundled by bip32 and bip39 packages on their own